pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -0,0 +1,42 @@
+= Applying Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Applying custom policies is the same as applying xref:gateway::policies-included-directory.adoc[included policies].
+
+When applying custom policies, the configuration parameters are the parameters defined in `definition/gcl.yaml`.
+
+To learn more about defining configuration parameters, see xref:policies-pdk-create-schema-definition.adoc[].
+
+== Before You Begin
+
+* xref:policies-pdk-publish-policies.adoc[Upload your Custom Policy to Exchange]
```

Comment:
Does this apply to Local Mode as well? If not you may want to preface this with some kind of "For Connected Mode, upload..."

Reply:
Moved to https://github.com/mulesoft/docs-gateway/pull/504/files